evalml/tests/automl_tests/parallel_tests/test_automl_dask.py

```diff
@@ -17,7 +17,7 @@
 
 # The engines to parametrize the AutoML tests over.  The process-level parallel tests
 # are flaky.
-engine_strs = ["cf_threaded", "dask_threaded"]
+engine_strs = ["dask_threaded"]
```

Comment:
why remove 'cf_threaded' here?

Reply:
This was for the previously failing tests which are fixed now, props to @jeremyliweishih for noticing this. I've included the reversion in my PR now